src/dataset/DatasetTree.ts

```diff
@@ -632,15 +632,18 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
         } else {
             dataSetName = node.getParent().getLabel();
         }
-        const afterMemberName = await vscode.window.showInputBox({value: beforeMemberName});
+        const afterMemberName = await vscode.window.showInputBox({ value: beforeMemberName });
```

Comment:
Could this PR be made simpler by just doing `toUpperCase()` at the point when `afterMemberName` is assigned in this function?

Reply:
Hi Lauren, thanks for reviewing my PR. 
Initially, we did just put the `toUpperCase()` method on the line that `afterMemberName` was assigned. However, when testing this, the following error occurred when cancelling the rename by pressing escape.

<img width="466" alt="Screenshot 2020-08-13 at 10 15 59" src="https://user-images.githubusercontent.com/43643821/90116869-0dfb9300-dd4e-11ea-8212-59ce9900642d.png">

This error occurred because the input box was returning nothing, hence why we changed it.

Do you think this error is significant enough to justify this approach?

Reply:
@AndrewTwydell Thanks for pointing out the case where the user cancelled via Escape!

Could we avoid running into this error by doing a check for `afterDataSetName` after it's assigned, and returning early out of the rename function if `afterDataSetName` is undefined? Then, if it's not undefined, we could convert `afterDataSetName` to uppercase and continue on with the rename function. For instance, something like:

```
let afterDataSetName = await vscode.window.showInputBox({value: beforeDataSetName});
if (!afterDataSetName) {
    vscode.window.showInformationMessage(localize("renameDataSet.cancelled", "Rename operation cancelled."));
    return;
}
afterDataSetName = afterDataSetName.toUpperCase();
... (the rest of the function...)
```

Would this work?

Reply:
Another great suggestion @lauren-li, thank you 😃 

However one of the many different implementations tried was very similar to this. The issue I found was that `afterDataSetName` was declared as constant, so could not then be changed to uppercase with `afterDataSetName = afterDataSetName.toUpperCase();` after I made sure it was populated.

Above, you have declared it using `let` as opposed to `const` which would solve this problem.

Do you recommend changing `afterDataSetName` to be a variable rather than constant?

Reply:
Hey @lauren-li , thanks for the suggestion, I tried this and it works! I will push the changes shortly 😃

Reply:
Thanks @Pranay154 !

@AndrewTwydell I think it should be okay to use `let` instead of `const` in this case. That gives us the flexibility to reassign `afterDataSetName` to the uppercase version if it's not undefined.

If there is a concern with using `let afterDataSetName`, you could use `const afterDataSetName` (as you did before), and then assign the value `afterDataSetName.toUpperCase();` to a differently-named constant that you could then use for the remainder of the rename function. However, I didn't see the case for doing that, so I went ahead and used `let afterDataSetName`.